.github/workflows/rubocop.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: rubocop
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: RuboCop Linter Action
+        uses: andrewmcodes/rubocop-linter-action@v3.3.0
```

Comment:
https://github.com/andrewmcodes/rubocop-linter-action#rubocop-linter-action states:
> PLEASE DO NOT USE THIS ACTION, and instead follow [the Rails way](https://github.com/rails/rails/blob/4a78dcb326e57b5035c4df322e31875bfa74ae23/.github/workflows/rubocop.yml#L1).

Reply:
Interesting, adopted Rails job. It's unfortunate not to see those nice comments but this will deliver a good experience too. It's super fast at least.

Reply:
The comments don't work on PRs from other repos, which is in practice all PRs.